jenkins/deployment/community/cortx-community-deploy.groovy

```diff
@@ -15,210 +14,204 @@ pipeline {
-
-                rm -rvf /usr/local/bin/aws /usr/local/bin/aws_completer /usr/local/aws-cli >/dev/null 2>&1
-                curl "https://awscli.amazonaws.com/awscli-exe-linux-x86_64.zip" -o "awscliv2.zip" && yum install unzip -y && unzip awscliv2.zip
-                ./aws/install
-                aws configure set default.region $REGION; aws configure set aws_access_key_id $ACCESS_KEY; aws configure set aws_secret_access_key $SECRET_KEY
-                pushd solutions/community-deploy/cloud/AWS
-                    ./tool_setup.sh
-                    sed -i 's,os_version =.*,os_version = "'"$OS_VERSION"'",g' user.tfvars && sed -i 's,region =.*,region = "'"$REGION"'",g' user.tfvars && sed -i 's,security_group_cidr =.*,security_group_cidr = "'"$VM_IP/32"'",g' user.tfvars && sed -i 's,ebs_volume_count =.*,ebs_volume_count = "'"$EBS_VOLUME_COUNT"'",g' user.tfvars && sed -i 's,ebs_volume_size =.*,ebs_volume_size = "'"$EBS_VOLUME_SIZE"'",g' user.tfvars
-                    echo key_name = '"'$KEY_NAME'"' | cat >>user.tfvars
-                    cat user.tfvars | tail -6
-                popd
+                export VOLUME_COUNT=${VOLUME_COUNT}
+                export VOLUME_SIZE=${VOLUME_SIZE}
+                export INSTANCE_COUNT=${INSTANCE_COUNT}
+                export INSTANCE_TAG_NAME=${INSTANCE_TAG_NAME}
+                            rm -rvf /usr/local/bin/aws /usr/local/bin/aws_completer /usr/local/aws-cli >/dev/null 2>&1
+                            curl "https://awscli.amazonaws.com/awscli-exe-linux-x86_64.zip" -o "awscliv2.zip" && yum install unzip -y && unzip awscliv2.zip
+                            ./aws/install
+                            aws configure set default.region $REGION; aws configure set aws_access_key_id $ACCESS_KEY; aws configure set aws_secret_access_key $SECRET_KEY
+                        pushd solutions/community-deploy/cloud/AWS
+                            ./tool_setup.sh
+                            sed -i 's,os_version          =.*,os_version          = "'"$OS_VERSION"'",g' user.tfvars && sed -i 's,region              =.*,region              = "'"$REGION"'",g' user.tfvars && sed -i 's,security_group_cidr =.*,security_group_cidr = "'"$VM_IP/32"'",g' user.tfvars && sed -i 's,instance_count          =.*,instance_count          = "'"$INSTANCE_COUNT"'",g' user.tfvars && sed -i 's,ebs_volume_count          =.*,ebs_volume_count          = "'"$VOLUME_COUNT"'",g' user.tfvars && sed -i 's,ebs_volume_size          =.*,ebs_volume_size          = "'"$VOLUME_SIZE"'",g' user.tfvars && sed -i 's,tag_name          =.*,tag_name          = "'"$INSTANCE_TAG_NAME"'",g' user.tfvars
+                            echo key_name            = '"'$KEY_NAME'"' | cat >>user.tfvars
+                            cat user.tfvars
+                            popd
                 '''
             }
         }
-
-                stage ('Create EC2 instace') {
+        stage('Create Multi EC2 instances') {
```

Comment:
we can use this job for 1-node deployment as well right? if yes, this stage name won't be suitable

Reply:
Can we update stage name to `Create EC2 instance`

Reply:
fixed